src/rpc/request.cpp

```diff
@@ -131,7 +134,10 @@ bool GetAuthCookie(std::string *cookie_out)
 void DeleteAuthCookie()
 {
     try {
-        fs::remove(GetAuthCookieFile());
+        if (g_generated_cookie) {
```

Comment:
nit: can be done with a bit less nesting

<details>
<summary>git diff on 7cb9367157</summary>

```diff
diff --git a/src/rpc/request.cpp b/src/rpc/request.cpp
index b7acd62ee3..ad9f244f12 100644
--- a/src/rpc/request.cpp
+++ b/src/rpc/request.cpp
@@ -133,11 +133,10 @@ bool GetAuthCookie(std::string *cookie_out)
 
 void DeleteAuthCookie()
 {
+    // Don't delete the cookie file if it wasn't generated by this process
+    if (!g_generated_cookie) return;
     try {
-        if (g_generated_cookie) {
-            // Delete the cookie file if it was generated by this process
-            fs::remove(GetAuthCookieFile());
-        }
+        fs::remove(GetAuthCookieFile());
     } catch (const fs::filesystem_error& e) {
         LogPrintf("%s: Unable to remove random auth cookie file: %s\n", __func__, fsbridge::get_filesystem_error_message(e));
     }

```
</details>